### PR TITLE
Update plugin to work with pytest 4.1.0

### PR DESCRIPTION
**Pull request by marcos-alfredo**
Hi,

The latest pytest update (4.1.0) removed the get_marker function, breaking the plugin. See comment on https://docs.pytest.org/en/latest/changelog.html for details.
The fix is just a single line replacement.

I wasn't sure if it was best to open an issue or this MR but since I fixed it for our project anyway, figure I'll send it your way and you can decide how to best go about it.

### CODE REVIEW

File: setup.py
Comment (schettino72) [inline setup.py]: Since this requires a major upgrade on py.test, we also need to bump our version number to `0.2.0`.
Or lets call it `1.0.0` :smile:

**Review by schettino72 — CHANGES_REQUESTED** [review]
Thanks. Pull-requests are always better than creating an issue...
Could you please also add an entry on `CHANGES` file?

Would you like to co-maintain this project? I am sending you github invitation.
Do you have a pypi account? I could also add you as a maintainer there so you can push a new release yourself...

File: setup.py
Comment (marcos-alfredo) [reply setup.py]: Sounds good, version changed.

**Comment by marcos-alfredo** [comment]
Added the new entry to the CHANGES file and updated the version.

I've never worked with pypi but it should be an interesting experience, so sure.
Just created an account there, username marcos.alfredo

I also pushed a small change to fix the tests but it only worked for Python 3.4, I'll try to take a look later into why.

Edit: okay, quick look so for 2.7 I'll try freezing the version of doit. For Python 3.3, pytest doesn't list it as supported version, so maybe we should drop it? Also maybe add 3.5, 3.6 and 3.7 to the CI script. I'll check tomorrow.

**Comment by schettino72** [comment]
Are you still using python2.7? if not, just drop it...

Yes, supported versions should be 3.5 to 3.7.

**Comment by schettino72** [comment]
Cool. I have just added you as a maintainer on PyPI.
Feel free to add your name somewhere on README file, or create an AUTHORS file to give credit to yourself.
Also feel free to merge changes and publish on PyPI :)

**Comment by marcos-alfredo** [comment]
I'm not using 2.7 but pytest still supports it and the plugin works, it was just a matter of fixing the tests on the CI pipeline.

It looks good now, I'll see about the authors thing.

After that should be good to merge and I'll figure out how to publish it.

**Comment by marcos-alfredo** [comment]
Okay, I think this is it.
Added my name to the readme and the setup.py file.
Changed a few others things there that I think make sense.
I've tested how to publish the plugin on the PyPI test site, you can take a look here https://test.pypi.org/project/pytest-ignore-flaky/.

If you agree with all changes we can go ahead with the merge and I'll publish it to the main site.

**Comment by schettino72** [comment]
perfect. thanks.
if you could upload to PyPI, it would be great.

**Pull request merged**
